modules/mtc-operator-supported.adoc

```diff
@@ -0,0 +1,43 @@
+// Module included in the following assemblies:
+//
+// * migration_toolkit_for_containers/about-mtc.adoc
+
+:_mod-docs-content-type: REFERENCE
+[id="mtc-operator-supported_{context}"]
+= Support for {mtc-first}
+
+.Supported versions of {mtc-short}
+[width="100%",cols="10%,14%,20%,22%,22%,22%,options="header"]
+|===
+|Version
+|{mtc-short} version
+|General availability
+|Full support ends
+|Maintenance ends
+|Extended Update Support (EUS)
+
+|1.8
+a|
+* 4.14
+* 4.15
+* 4.16
+|02 Oct 2023
+|Release of 1.9 
+|Release of 1.10
+|N/A  
+
+|1.7
+a|
+* 3
+* 4.12
+* 4.13
+* 4.14
+* 4.15
```

Comment:
```suggestion
|1.7
a|
* 3
* 4.14
* 4.15
* 4.16
```

We should ensure alignment with what's stated [here](https://access.redhat.com/support/policy/updates/openshift_operators#platform-agnostic)
![image](https://github.com/user-attachments/assets/618f01b3-7153-41e2-87f7-86c001634b65)